README.md

```diff
@@ -93,60 +102,50 @@ registrati:
 
 The application currently supports the following code hosting platforms:
 
-* https://github.com
+* [https://github.com](https://github.com)
 
-* https://bitbucket.org
+* [https://bitbucket.org](https://bitbucket.org)
 
-* https://gitlab.com
+* [https://gitlab.com](https://gitlab.com)
 
-* https://phabricator.com
+* [https://phabricator.com](https://phabricator.com)
 
-* https://gitea.io
+* [https://gitea.io](https://gitea.io)
 
-* https://gogs.io
+* [https://gogs.io](https://gogs.io)
 
 Stay tuned. The list keeps growing...
```

Comment:
I don't know if this makes still sense.. As of now there is no validation on code hosting platform on this side but it is still true that the crawler doesn't support few of them.

cc @libremente

Reply:
@sebbalex Agreed, the parser repo is a better place for this IMO.

Reply:
@sebbalex I removed that part.